outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
```

Comment:
That's a great idea—I like the structure of motivations –> obstacles –> solutions –> examples/case studies